File: week 1/mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height , i, j;

     //  do while loop to get height by user input between 1 to 8
    do
    {
        height = get_int("Height: ");
    }while((height<1) || (height>8));

     // double for loop to make the double pyramid for mario
    for(i = 1;i<=height;i++)
    {
        for(j = 1; j <= ((height*2) + 2) ;j++)
        {
            if((j == height+1) || (j == height+2))            {
                printf(" ");
            }
            else if(j <= height)
            {
                 if((j<=height) && (j>=height+1-i))
                    {
                    printf("#");
                    }
                 else{
                        printf(" ");
                    }

            }
            else if(j>height+2)
            {
                if(j<=(i+height+2))
                {
                    printf("#");
                }
            }

        }
        printf("\n");
    }
}

File: week 6/sentimental-mario-more/mario.py
from cs50 import get_int
height = 0
while height > 8 or height < 1:
    height = get_int("Height: ")

# Loop through rows and then columns
for i in range(1, height + 1):
    for j in range(1, height + 1):
        # Start printing # after correct number of spaces
        if j > (height - i):
            print("#", end="")
        else:
            print(" ", end="")
        # Print middle space and remaining #
        if j == height:
            print(" ", "#" * i, end="")
    print()



# while(True):
#     try:
#         height = int(input("enter the height: "))
#         if height > 0 and height < 9:
#             break
#     except:
#         print("invalid input")



# for i in range(height):
#     for j in range((height*2)+2):
#         if j == height or j == height+1:
#             print(" ",end="")
#         elif j < height:
#             if j<height and j>=height-1-i:
#                 print("#", end="")
#             else:
#                 print(" ",end="")
#         elif j >= height+1:
#             if j<=i+height+2:
#                 print("#", end="")
#             else:
#                 print(" ", end="")
#     print()
